magicctapipe/scripts/lst1_magic/semi_automatic_scripts/DL1_to_DL2.py

```diff
@@ -137,6 +142,8 @@ def bash_DL1Stereo_to_DL2(target_dir, source, env_name, cluster, RF_dir, df_LST,
             dec = df_LST[df_LST.source == source].iloc[0]["MC_dec"]
             dec = str(dec).replace(".", "")
             RFdir = f"{RF_dir}/{period}/NSB{nsb}/{MC_v}/dec_{dec}/"
```

Comment:
if you want you could make a shorthand if statement instead:
```
RFdir = f"{RF_dir}/{period}/NSB{nsb}/{MC_v}/dec_{dec}{'_high_density' if source in dense_list else ''}/"
````
it avoids code duplication and managing the same path in two places, but the line gets more complicated, so if you prefer the original it is also perfectly fine with me